oaknational/Oak-Web-Application
# [CUR-917] Close button on lot picker for subject phase

## Description

Music year: 1951

- Add a close button to the lot picker for both subject and phase modals

## How to test

1. Go to https://deploy-preview-2951--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Click on the subject selector
3. You should see the close button in the top right corner
4. Click on the button and the modal should close
5. Repeat for phase selector
6. Shouldn't impact subject or phase selection

## Should look like
<img width="1059" alt="Screenshot 2024-10-31 at 15 40 45" src="https://github.com/user-attachments/assets/5fd9049c-6011-4178-b401-7e9d4bbe562b">
<img width="613" alt="Screenshot 2024-10-31 at 15 40 55" src="https://github.com/user-attachments/assets/4e90a0da-448e-4eee-bd68-217416b3f8a7">

## Checklist
